.github/workflows/testbuild.yml

```diff
@@ -9,42 +14,47 @@ env:
 jobs:
   build:
     name: Build
-    # The CMake configure and build commands are platform agnostic and should work equally
-    # well on Windows or Mac.  You can convert this to a matrix build if you need
-    # cross-platform coverage.
-    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
     runs-on: ubuntu-latest
 
     steps:
     - name: Checkout Repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         submodules: recursive
-    
-    - name: Build moonjit
+
+    - name: Set up GCC
+      uses: egor-tensin/setup-gcc@v1
+      with:
+        version: latest
+        platform: x64
+
+    - name: Build libopus
+      shell: bash
+      working-directory: ${{github.workspace}}/opus
```

Comment:
Why are we building opus instead of installing the package.

Reply:
The package is not usually updated by package managers. If I remember correctly, jdb has libopus as a submodule on git and it was referenced in the cmake.